addons/website_livechat/static/src/models/visitor/visitor.js

```diff
@@ -113,6 +113,10 @@ function factory(dependencies) {
                 'partnerCountry',
             ],
         }),
+        /**
+         * Name of the visitor.
```

Comment:
display name now I guess?

Reply:
Distinction is important because it is one of those models where `name_get` is actually overriden.